src/mats-monitor-hrly-value-workspace/mats-monitor-hrly-value.repository.ts

```diff
@@ -0,0 +1,17 @@
+import { EntityRepository, Repository } from 'typeorm';
+import { MatsMonitorHrlyValue } from '../entities/workspace/mats-monitor-hrly-value.entity';
+
+@EntityRepository(MatsMonitorHrlyValue)
+export class MatsMonitorHrlyValueRepository extends Repository<
+  MatsMonitorHrlyValue
+> {
+  async export(
+    monitoringLocationIds: string[],
+  ): Promise<MatsMonitorHrlyValue[]> {
+    return this.createQueryBuilder('MatsMonitorHrlyValue')
+      .where('MatsMonitorHrlyValue.mon_loc_id IN (...:monitoringLocationIds', {
```

Comment:
Where condition should check where hour id is the same as the id in hourly operating data